src/passes/OptimizeInstructions.cpp

```diff
@@ -258,9 +254,9 @@ struct OptimizeInstructions
             //    zero is impossible to be equal to a sign-extended value
             //    anyhow, so the entire thing is false.
             //  * if they were all one, we may get a false false, if the only
-            //    difference is in those upper bits. that means we are equal on
-            //    the other bits, including the sign bit. so we can just mask
-            //    off the upper bits in the constant value, in this case,
+            //    difference is in those upper bits. that means we are equal
+            //    on the other bits, including the sign bit. so we can just
+            //    mask off the upper bits in the constant value, in this case,
```

Comment:
There are still a few unrelated comment changes here and above that would be good to remove.

Reply:
done!